src/main/java/kr/codesquad/LadderGameController.java

```diff
@@ -0,0 +1,14 @@
+package kr.codesquad;
+
+public class LadderGameController {
+    public static void run() {
+        final int participants = View.readParticipants();
+        final int ladderHeight = View.readLadderHeight();
+
+        Ladder ladder = new Ladder(participants, ladderHeight);
```

Comment:
```suggestion
        Ladder ladder = new Ladder( View.readParticipants(), View.readLadderHeight());
```

이렇게도 할 수 있겠죠? 지역변수도 적게 사용하는 편이 좋다고 합니다.